infra/src/main/kotlin/fi/fta/geoviite/infra/geometry/GeometryDao.kt

```diff
@@ -226,20 +226,20 @@ class GeometryDao @Autowired constructor(
             with
               location_tracks as (
                 select
-                  distinct ga.plan_id, coalesce(track.draft_of_location_track_id, track.id) as id
+                  distinct ga.plan_id, coalesce(track.official_row_id, track.id) as id
```

Comment:
Vähän rupesin miettimään tätä tehdessä vielä näitä nimeämisiä... Menin edelleen alunperin speksatulla nimimuutoksella, mutta on mun mielestäni jotenkin outoa, että tämän jäljiltä paikannuspohjan virallisilla käsitteillä `official_row_id`-columni on tyhjä. Aiempi `draft_of_x_id`-nimi kuvasi osittain sitä että tämä on ainoastaan drafteilla olevaa tietoa, mitä tämä nykyinen ei mielestäni tee.

Reply:
Se on totta. Tuo on kuitenkin konseptin laajentuessa minusta selkeämpi. Jatkossa on siis vaihtoehdot:
-  official(draft=false, row_id)
- draft(draft=true, row_id, official_row_id, design_row_id)
- design(draft=false, row_id, official_row_id)
- design-draft(draft=true, row_id, official_row_id, design_row_id)

Jos tuota käyttötarkoitusta kuvaavammat nimet (draft_of) ottaisi mukaan niin oikeastaan nuo kentät ei enää riittäisi koska on eri asiat design_of_x ja draft_of_x ja draft_of_design_of_x, jne. Se räjähtää käsiin, vaikka oikeasti riittää 3 id:tä (oma, virallisen rivin id, design-rivin id)